src/dbnode/storage/shard.go

```diff
@@ -804,7 +804,7 @@ func (s *dbShard) writeAndIndex(
 		// Retrieve the inserted entry
 		entry, err = s.writableSeries(id, tags)
```

Comment:
I think technically just because the series appeared here doesn't actually mean the write succeeded. In practice, I don't think it matter, but maybe worth a comment. (You can imagine a situation where two writes for the same series come in and get batched together. The first one fails (due to some issue in the buffer) and the second one succeeds. Both write will "look" like they succeeded here, but you don't actually know if the buffer rejected it or not for being a dual-write

Reply:
is this no longer relevant? @arnikola Not sure what I wrote this in reference to

Reply:
I'm not super sure what it's referring to, sorry... Either way I think it should be fine since worst case scenario in this case is that this series will be written twice to the commit log, right?